### ISSUE TITLE
Halls of Stone - Tribunal Of Ages

### ISSUE THREAD

**Issue by thestranger064** [dalaranwow/dalaran-wow#5492]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: During Tribunal Of Ages , half the script is missing. The orbs, lazer and the periodic damage over time never happens, making the entire encounter revolve around just killing adds, which makes it a bit boring.

**Expected behaviour**: The Tribunal Of Ages is an event in Halls of Stone where you must defend Brann. Waves of mobs will come out at you for 5 minutes and you must kill them before they reach Brann, or even damage Brann if you want the achievement. In addition a laser will also activate in the ceiling. This laser will go around the room and do damage to anyone it passes over. Below is a list of the timers for this event, credit goes to Rossogia of Wowhead.com.

0:00 Start of the Event
**0:45 Enter Combat the weak bolts that don't really do much begin to fire.**
0:50 First Wave will start spawning (Dark Rune Protectors)
1:40 The next "Cutscene" beings
**1:45 First Dark Matter Orbs and Shadow Word:Pain beings** 
2:00 Second mob type begins spawning in addition to first (2x Dark Rune Stormcallers)
3:20 The next "Cutscene" begins"
**3:25 Searing Gaze attacks begin**
3:45 First Iron Golem Custodian spawns
5:00 Final "cutscene" starts
5:15 You win!

**Steps to reproduce the problem**:

1. Get into the dungeon Halls of Stone
2. Activate Tribunal Of Ages
3. See that it is missing it script.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Feedback on #2049
